{
	"entries": [
		{
				"message": "Running ApiTest1.apitest",
				"identifier": "START",
				"timestamp": "2025-02-26 13:18:22.591 +0530"
			},
		{
				"message": "(Duration:\"3.057\", Errors:\"1\", Exceptions:\"0\", StartTime:\"2025-02-26 13:18:22 +0530\", Successes:\"0\", TestCase:\"ApiTest1.apitest\", Warnings:\"0\")",
				"identifier": "EndTestCase",
				"timestamp": "2025-02-26 13:18:25.656 +0530"
			},
		{
				"image": "Screen_Error.png",
				"message": "STMaxCallDepthExceeded: Handlers were called to a depth exceeding the maximum limit (40). HINT: One common cause for this error is a script that has been given the same name as a built-in command or function unintentionally calling itself. -- Execution Time 0:00:03 ApiTest1.apitest",
				"identifier": "FAILURE",
				"timestamp": "2025-02-26 13:18:26.366 +0530"
			}
	],
	"createdTimestamp": "2025-02-26 13:18:22.596 +0530"
}